robot_controllers_interface/src/controller_loader.cpp

```diff
@@ -46,11 +46,17 @@ bool ControllerLoader::init(const std::string& name, ControllerManager* manager)
 
   if (nh.getParam("type", controller_type))
   {
-    controller_ = plugin_loader_.createInstance(controller_type);
-    controller_->init(nh, manager);
+    // If plugin is bad, catch pluginlib exception 
+    try
+    {
+      controller_ = plugin_loader_.createInstance(controller_type);
+      controller_->init(nh, manager);
+    }catch( pluginlib::LibraryLoadException e)
```

Comment:
we follow the ROS style guide -- "catch (pluginlib::LibraryLoadException e)" should be on it's own line and should have spacing as noted in this comment